src/BuildConfig.cc

```diff
@@ -541,6 +603,10 @@ BuildConfig::setVariables() {
   for (const std::string_view flag : profile.cxxflags) {
     cxxflags.emplace_back(flag);
   }
+  for (const std::string& flag : getEnvFlags("CXXFLAGS")) {
```

Comment:
Let's add a comment of why `CXXFLAGS` will be appended at last; it's because environment variables take the highest precedence, right? (please also add a similar comment right above `LDFLAGS` as well)

Reply:
Fixed in the latest commit.